models/CoronSegmentator/configs/metadata.json

```diff
@@ -0,0 +1,88 @@
+    "description": "Coronsegmentator is an automated pipeline that performs dual-task segmentation on cardiac CT images, focusing on both whole-heart and coronary artery structures. It integrates MONAI Auto3DSeg for general cardiac segmentation and a custom nnU-Net model for detailed coronary artery segmentation. The pipeline further converts segmentation results (in STL format) into USD files for downstream 3D visualization and digital twin simulation using NVIDIA Omniverse.",
+    "authors": "Y. Ke, MC. Chen, TY. Lin, YC. Chan Foxconn Digital Health AI Team",
+    "copyright": "Copyright \u00a9 2025 Hon Hai Precision Industry Co.,Ltd. All rights reserved",
+    "data_source": "The ImageCAS dataset is publicly available at: https://github.com/XiaoweiXu/ImageCAS-A-Large-Scale-Dataset-and-Benchmark-for-Coronary-Artery-Segmentation-based-on-CT.git",
+    "data_type": "nifti",
+    "image_classes": "3D volume data",
+    "label_classes": "single channel data, 1 is coronary artery, 0 is background",
+    "pred_classes": "2 channels OneHot data, channel 1 is coronary artery, channel 0 is background",
+    "intended_use": "This is an example, not to be used for diagnostic purposes",
+    "references": [
+        "Zeng, An, et al. ImageCAS: A large-scale dataset and benchmark for coronary artery segmentation based on computed tomography angiography images. Computerized Medical Imaging and Graphics 109 (2023): 102287.",
+        "Isensee, Fabian, et al. nnU-Net: a self-configuring method for deep learning-based biomedical image segmentation. Nature methods 18.2 (2021): 203-211."
+    ],
+    "network_data_format": {
+        "inputs": {
+            "image": {
+                "type": "image",
+                "format": "hounsfield",
+                "modality": "CT",
+                "num_channels": 1,
+                "spatial_shape": [
+                    512,
+                    512,
+                    256
+                ],
+                "dtype": "float32",
+                "value_range": [
+                    0,
+                    1
+                ],
+                "is_patch_data": true,
+                "channel_def": {
+                    "0": "image"
+                }
+            }
+        },
+        "outputs": {
+            "pred": {
+                "type": "usd",
+                "format": "segmentation",
+                "num_channels": 1,
+                "spatial_shape": [
+                    512,
+                    512,
+                    256
+                ],
+                "dtype": "float32",
+                "value_range": [
+                    0,
+                    1
+                ],
+                "channel_def": {
+                    "0": "Background",
+                    "1": "Coronary Artery"
+                }
+            }
+        }
+    },
+    "optional_packages_version": {
+        "monai": "1.3.2",
+        "torch": "2.7.0+cu126",
+        "numpy": "2.3.1"
+    }
```

Comment:
I guess you can remove this part, the "optional_packages_version" is no longer a required one in the latest schema. thanks.